consensus/polybft/state.go

```diff
@@ -59,6 +62,8 @@ const (
 	stateSyncMainBundleSize = 10
 	// number of stateSyncEvents to be grouped into one StateTransaction
 	stateSyncBundleSize = 1
+	// static key which is used to save latest proposer snapshot. It will be always one object in bucket
+	proposerSnapshotStaticKey = 7
```

Comment:
The static key should be a `[]byte` not a number.

Reply:
@ferranbt Seems it is impossible to put empty byte slice as a key. Tried it and I'm getting an error from bolt db ("key required").

I agree that it is strange we have 7 as a key, but not sure what to do here? Should we use `proposerSnapshot.Height | proposerSnapshot.Round` or maybe just `proposerSnapshot.Height` and then fetch it using `cursor.Last()/cursor.First()` (although I guess that would probably have _some_ performance penalty)...

Reply:
Why not just use `proposerSnapshotKey = []byte("oneKeyToRuleThemAll")`. Just some string, if we are always rewriting that one snapshot.

Reply:
Fixed per @goran-ethernal suggestion (https://github.com/0xPolygon/polygon-edge/pull/1018/commits/184be592c0d4fd3e7f18d786152fc1c73a1890ba)